gladier_kanzus/tools/gather_data.py

```diff
@@ -100,13 +104,7 @@ def ssx_gather_data(**data):
     data['pilot']['metadata'] = metadata
 
     return {
-        'pilot': data['pilot'],
-        'plot': {
```

Comment:
You still get plot info from beamline.json on the old version. Why it cannot be acquired directly into the plot script?

Reply:
The gather metadata function previously fetched the beamline.json stuff in order to include it in the metadata. The key pieces of info were then passed to the plotting script through flows. It made for a bit more complexity when configuring the flow, but it also meant that the plotting script didn't need to know anything about beamline.json. If anything about the beamline.json file changed, we wouldn't need to update the plotting function. This draws from the design principle here: 
https://en.wikipedia.org/wiki/Separation_of_concerns